### ISSUE TITLE
Emojis

### ISSUE THREAD

**Issue by dori921** [surfranch/TelegramShillBot#42]
Hello.Thanks for this great script.
Is there a way i can add emojis on yaml file ?

**Comment by gkspranger** [comment]
yes -- i believe most people just copy/paste them into the yaml file itself and we have seen this work
